doc/modules/experimental.rst

```diff
@@ -0,0 +1,41 @@
+.. _experimental_ref:
+
```

Comment:
should we just copy the "header" from scikit learn for this module?

https://scikit-learn.org/stable/modules/classes.html#module-sklearn.experimental

Reply:
Done